### DISCUSSION TITLE
token design体系如何 处理渐变色背景，渐变色边框等样式

### DISCUSSION THREAD

**Discussion by zhuxiangjian** [ant-design/ant-design#38945] [General]
如题，比如UI设计了基于主题色的 渐变色primary  button该怎么优雅的修改样式，另外对于边框样式设置渐变边框色该如何处理。对于各种类型Button不显示边框线样式能否用token design体系做到。

**Comment by arvinxx** [comment]
暂时不支持，后面会给方案，这个是在规划之内的。
